applications/potree-converter/app.json

```diff
@@ -0,0 +1,91 @@
+{
+    "id": "potree-converter",
+    "version": "2.1.1",
+    "description": "Convert pointclouds to Potree format.",
+    "owner": "${apiUserId}",
+    "enabled": true,
+    "runtime": "ZIP",
+    "runtimeVersion": null,
+    "runtimeOptions": null,
+    "containerImage": "tapis://cloud.data/corral/tacc/aci/CEP/applications/v3/potree-converter/potree-converter.zip",
+    "jobType": "FORK",
+    "maxJobs": -1,
+    "maxJobsPerUser": -1,
+    "strictFileInputs": true,
+    "jobAttributes": {
+        "description": "",
+        "dynamicExecSystem": false,
+        "execSystemConstraints": null,
+        "execSystemId": "wma-dcv-01",
```

Comment:
I wonder if using `wma-dcv-01` could be a problem as it has 32GB of memory.  Some large las files could use 50gb of memory (we have have hit this in hazmapper so our backend worker has 100 GB of memory and we have plans to limit the amount of simulation conversion of large las files).

I'm not certain if this would be a problem at all here. but something to consider I guess for all apps that share wma-dcv-01.  maybe, we just need to ask for more memory on that machine.